constants.go

```diff
@@ -431,9 +431,9 @@ const (
 	// deadline in cases where both require_session_mfa and disconnect_expired_cert
 	// are enabled. See https://github.com/gravitational/teleport/issues/18544.
 	CertExtensionPreviousIdentityExpires = "prev-identity-expires"
-	// CertExtensionClientIP is used to embed the IP of the client that created
+	// CertExtensionLoginIP is used to embed the IP of the client that created
 	// the certificate.
-	CertExtensionClientIP = "client-ip"
+	CertExtensionLoginIP = "login-ip"
```

Comment:
I assume this rename is safe since we don't actually consume this value anywhere yet (but would be breaking if it were consumed) ?

Reply:
Yes, that's the case, as aslo noted by Roman ( https://github.com/gravitational/teleport/pull/21866#discussion_r1107455949 )